management_api_app/tests_ma/test_api/test_routes/test_workspace_service_templates.py

```diff
@@ -139,6 +161,41 @@ async def test_when_creating_workspace_service_template_service_resource_type_is
         create_workspace_template_item_mock.assert_called_once_with(input_workspace_template,
                                                                     ResourceType.WorkspaceService)
 
+    # GET /workspace-service-templates/{template_name}
+    @patch("api.routes.workspace_service_templates.ResourceTemplateRepository.get_current_resource_template_by_name")
+    async def test_workspace_service_templates_by_name_returns_enriched_workspace_service_template(self, get_workspace_service_template_by_name_mock,
+                                                                                                   workspace_service_template_without_enriching,
+                                                                                                   app: FastAPI, client: AsyncClient):
+        template_name = "template1"
+        get_workspace_service_template_by_name_mock.return_value = workspace_service_template_without_enriching(template_name)
+
+        response = await client.get(app.url_path_for(strings.API_GET_WORKSPACE_SERVICE_TEMPLATE_BY_NAME, template_name=template_name))
+
+        assert response.status_code == status.HTTP_200_OK
+        assert response.json()["name"] == template_name
+        assert "description" in response.json()["required"]
+
+    # GET /workspace-service-templates/{template_name}
+    @patch("api.routes.workspace_service_templates.ResourceTemplateRepository.get_current_resource_template_by_name")
```

Comment:
could combine the test below this with parametrized tests. Make it DRY

Reply:
fixed